ee/modules/380-metallb/templates/speaker/bgp-advertisements.yaml

```diff
@@ -0,0 +1,10 @@
+{{- range $ipaddressPool := .Values.metallb.addressPools }}
+  {{ if eq $ipaddressPool.protocol "bgp" }}
+---
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
```

Comment:
weird cr without specs :)

Reply:
Oh, it is deprecated one.

Reply:
discussed, this is ok. Without spec bgp advertises all address pools